### DISCUSSION TITLE
打开主页，仅显示文章的标题，不显示文章的概述、概要，是怎么回事耶

### DISCUSSION THREAD

**Discussion by Justsenger** [hugo-next/hugo-theme-next#92] [Q&A] [answered]
😃

**Comment by Justsenger** [comment]
如果开启expand: true的话会展开文章全部内容，我希望只展开一小段

**Comment by elkan1788** [comment, marked as answer]
尝试在文章中加个手动截断标记试试：`<!--more-->`，应该可以的。

**Reply by Justsenger** [reply]
> 尝试在文章中加个手动截断标记试试：`<!--more-->`，应该可以的。

发现提问前就已经有了...

**Reply by Justsenger** [reply]
这个截断标记

**Reply by elkan1788** [reply]
把你的站点仓库地址分享下，理论上不会出现这样的问题

**Reply by Justsenger** [reply]
> 把你的站点仓库地址分享下，理论上不会出现这样的问题

我找到原因了，我把截断标记放前面去了，没注意到应该放在最后。最后我的做法是把截断标记删掉，然后添加expand: false就能达到效果了😃

**Reply by Justsenger** [reply]
准确来说也不是放最后，放最后的话会默认提取文章所有内容，可以在---后面写概要，然后在后面加上截断标记，就完美了。
